packages/bp/src/core/events/event-repository.ts

```diff
@@ -30,6 +31,15 @@ export class EventRepository {
     let query = this.database.knex(this.TABLE_NAME)
     query = query.where(fields)
 
+    if (params.createdOn) {
+      if (params.createdOn.from) {
+        query = query.andWhere(this.database.knex.date.isAfterOrOn('createdOn', params.createdOn.from))
+      }
+      if (params.createdOn.to) {
+        query = query.andWhere(this.database.knex.date.isBeforeOrOn('createdOn', params.createdOn.to))
+      }
+    }
```

Comment:
why not use after and before instead of from and to ? really confusing keywords in this case

Reply:
Sure, i will change the naming, no issues on my side